src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(int radius) {
+        this.radius = radius;
```

Comment:
```suggestion
    public Circle(String color, int radius) {
        super(color);
        this.radius = radius;
```
lets also add field color